crates/runtime/src/http/routes.rs

```diff
@@ -67,6 +67,8 @@ pub(crate) fn routes(
             .route("/v1/predict", post(v1::inference::post))
             .route("/v1/nsql", post(v1::nsql::post))
             .route("/v1/embed", post(v1::embed::post))
+            .route("v1/oai/:name/chat/completions", post(v1::oai::chat))
```

Comment:
Let's make this purely compatible.

```suggestion
            .route("/v1/chat/completions", post(v1::oai::chat))
```

Reply:
I think we want to have a `:name` parameter so that we enable future compatibilies. It'll just mean setting the endpoint to be `http://localhost:300/oai/my_llm_name/`. But you are right, it'd have to be switched around.